src/Sulu/Bundle/AdminBundle/Metadata/FormMetadata/FormMetadataProvider.php

```diff
@@ -55,11 +55,31 @@ public function getMetadata(string $key, string $locale, array $metadataOptions
             foreach ($form->getForms() as $formType) {
                 $this->evaluateFormItemExpressions($formType->getItems());
             }
+
+            if (array_key_exists('tags', $metadataOptions)) {
+                $tags = $metadataOptions['tags'];
+                $tagAttributes = $metadataOptions['tagAttributes'] ?? [];
```

Comment:
this does not look intuitive to me.. shouldnt it be possible to filter for tags without a specific value?

Reply:
I think that might be a valid use case. And I have to agree that I have a hard time understanding this code :confused: